cob_bringup/robots/raw3-1.xml

```diff
@@ -31,19 +31,19 @@
 			<!--include file="$(find cob_bringup)/tools/wifi_monitor.launch">
 				<arg name="robot" value="$(arg robot)"/>
 			</include-->
-			<include file="$(find cob_bringup)/drivers/phidgets.launch">
+			<!--include file="$(find cob_bringup)/drivers/phidgets.launch">
 				<arg name="robot" value="$(arg robot)"/>
 			</include>
 			<include file="$(find cob_bringup)/drivers/powerstate_phidget.launch">
 				<arg name="robot" value="$(arg robot)"/>
 			</include>
 			<include file="$(find cob_bringup)/drivers/emstate_phidget.launch">
 				<arg name="robot" value="$(arg robot)"/>
-			</include>
-			<!-- Relayboard with sim:=True is necessary because the robot does not have a phidgets board for checking the emergency status -->
+			</include-->
```

Comment:
So, then just to re-confirm: 
 - `raw3-1` indeed does not have **any** phidget-sensor onboard
 - and the robot still works for you with these lines commented out?

If both questions are answered `yes`, then you could also remove the lines completely :wink: